### DISCUSSION TITLE
SemiAutomaticClassificationPlugin v. 7.10.6 - Error import signatures from shp.

### DISCUSSION THREAD

**Discussion by hemtopo** [semiautomaticgit/SemiAutomaticClassificationPlugin#214] [Q&A] [answered]
Hi, when I tried to import signatures from shape file, I received the following error: "File "C:\Users/PC GAMER/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\SemiAutomaticClassificationPlugin\core\utils.py", line 1893, in importShapefile
              trs = cfg.qgisCoreSCP.QgsCoordinateTransform(pCrs, tCrs)...".
I tried using versions of QGIS 3.22 and 3.16, all my information are in the same CRS.  I send more information and files to work.
[CorrecionAtmosferica.zip](https://github.com/semiautomaticgit/SemiAutomaticClassificationPlugin/files/8375646/CorrecionAtmosferica.zip)
[SCP_Error_Python.txt](https://github.com/semiautomaticgit/SemiAutomaticClassificationPlugin/files/8375647/SCP_Error_Python.txt)
![SCP1](https://user-images.githubusercontent.com/102632607/160724222-f92577d1-5ec6-4913-9f25-2893e8c1574c.jpg)
![SCP2](https://user-images.githubusercontent.com/102632607/160724223-ed757e63-9bc6-42bd-bf6d-230a99f4860c.jpg)
[FirmasDiss.zip](https://github.com/semiautomaticgit/SemiAutomaticClassificationPlugin/files/8375649/FirmasDiss.zip)

Thanks for your help.

Best regards.

**Comment by semiautomaticgit** [comment]
Hello @hemtopo , please copy the log file following these steps https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#how-can-i-report-an-error
Also, please try to move the input files in a directory without special characters such as C:\

**Reply by hemtopo** [reply]
Hi Luca, I send the log file, thanks.
[SCP_Debug.txt](https://github.com/semiautomaticgit/SemiAutomaticClassificationPlugin/files/8414278/SCP_Debug.txt)

**Reply by semiautomaticgit** [reply]
Hi @hemtopo , it could be an issue related to file path. 
Please try to move the input files to a path without accented characters

**Reply by hemtopo** [reply]
Thanks Luca, I moved information for a new path without accented characters, but the result is the same. Attach image.
![SCP3](https://user-images.githubusercontent.com/102632607/162551432-9680b123-81e4-4427-bd6e-5cf66ff6b165.jpg)

**Comment by semiautomaticgit** [comment, marked as answer]
Hello, sorry for the late reply. 
The cause of the issue should be the different coordinate reference systems of images and vector data. Please reproject the vector before importing it.
